src/main/java/uk/gov/justice/hmpps/prison/web/filter/UserMdcFilter.java

```diff
@@ -48,21 +38,20 @@ public void init(final FilterConfig filterConfig) {
     public void doFilter(final ServletRequest request, final ServletResponse response, final FilterChain chain)
             throws IOException, ServletException {
 
-        final var currentUsername = userSecurityUtils.getCurrentPrincipal();
+        final var authenticationOrNull = hmppsAuthenticationHolder.getAuthenticationOrNull();
 
         try {
-            if (currentUsername != null) {
-                MDC.put(USER_ID_HEADER, currentUsername);
-                telemetryClient.getContext().getUser().setId(currentUsername);
+            if (authenticationOrNull != null) {
+                MDC.put(USER_ID_HEADER, authenticationOrNull.getPrincipal());
+                telemetryClient.getContext().getUser().setId(authenticationOrNull.getPrincipal());
             }
-            telemetryClient.getContext().getComponent().setVersion(getVersion());
```

Comment:
don't need this line - the `applicationinsights.json` injects the version these days